assets/dev/scss/frontend/general/_global.scss

```diff
@@ -1,5 +1,5 @@
 // Default Hover Transition.
-.elementor-element:not(.e-div-block-base) {
+.elementor-element:where(:not(.e-con)):where(:not(.e-div-block-base)) {
```

Comment:
This seems to be on old bug. The general styling caused the container styling to break.

We used this CSS variable: `--e-transform-transition-duration`.
But for containers we need: `--e-con-transform-transition-duration`.